easybuild/easyconfigs/g/gmsh/gmsh-4.11.1-foss-2022a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'gmsh'
+version = '4.11.1'
+
+homepage = 'https://gmsh.info/'
+description = "Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://%(name)s.info/src/']
+sources = ['%(name)s-%(version)s-source.tgz']
+checksums = ['c5fe1b7cbd403888a814929f2fd0f5d69e27600222a18c786db5b76e8005b365']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('SWIG', '4.0.2'),
+    ('Eigen', '3.4.0'),
+]
+dependencies = [
+    ('Python', '3.10.4'),
+    ('PETSc', '3.17.4'),
+    ('SLEPc', '3.17.2'),
+    ('FLTK', '1.3.8'),
+    ('occt', '7.5.0p1'),
+]
+
+configopts = "-DENABLE_BUILD_SHARED=ON -DENABLE_WRAP_PYTHON=ON -DENABLE_METIS=1"
+
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s', 'bin/onelab.py', 'lib/%(name)s.py', 'lib/libgmsh.so'],
```

Comment:
Please use `SHLIB_EXT` instead of hardcoding `.so`

Reply:
Ah, right, will do. The `--try-*`-automation seems to have expanded `SHLIB_EXT` when parsing the file so when recreating the file it used the expanded value instead. That's annoying. I'll keep that in mind, thanks!

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/17534/commits/f7cb9a083ab4b051084c37027ff0839b76c65500